app/javascript/components/users/SignInComponent.tsx

```diff
@@ -57,47 +78,186 @@ function SignInComponent(props:SignInComponentProps) : JSX.Element {
 		}
 	}, [isValid, componentState]);
 
+
+	//Error messages
 	const { formatMessage } = useIntl();
+	const label = formatMessage({ id: 'email', defaultMessage: '* Requiered' });
+
+
+	//Yup validation
+	const validationSchema = yup.object({
+		email: yup.string().email().required(),
+		password: yup.string().required(),
+	});
+
+	//Styling
+	const useStyles = makeStyles((theme: Theme) => createStyles({
+		textField: {
+			'& .MuiTextField-root': {
+				margin: theme.spacing(1),
+			},
+		},
+		paper: {
+			maxWidth: 400,
```

Comment:
Does max-width 400 work well on small displays? I don't see why it wouldn't but  just wanted to know that it's been tested.

Reply:
Any update on this?

Reply:
Hi, I haven't heard back here.